packages/opentelemetry-plugin-http/test/functionals/http-enable.test.ts

```diff
@@ -48,7 +51,7 @@ function doNock(
     .reply(httpCode, respBody);
 }
 
-export const customAttributeFunction = (span: Span): void => {
+export const customAttributeFunction = (span: ISpan): void => {
```

Comment:
What is this for? Is this to convey it is being used as an interface only?

Reply:
yes, it's conflicting with the class and I need to do:
```
assert.strictEqual((span as Span).toReadableSpan()
```

Reply:
What will happen if we just keep `Span` instead of `ISpan` alias ? Compilation error ?

Reply:
because of this test : https://github.com/open-telemetry/opentelemetry-js/pull/244/files#diff-9b663d16d799f5f4830dbfb0c9d012f1R265 he's commented because of https://github.com/open-telemetry/opentelemetry-js/issues/226 but ultimately, it will work and I need to cast span as Span (the class).

Let me know.

Reply:
ok